kola/tests/misc/tpm.go

```diff
@@ -47,6 +54,127 @@ systemd:
         ExecStart=rm /etc/luks/rootencrypted
         [Install]
         WantedBy=multi-user.target
+`)
+	// Note: Keep the two below configs in sync with those
+	// documented in the Flatcar TPM docs.
+	// Ideally the reboot wouldn't be needed (or done in the initrd?)
+	IgnitionConfigRootCryptenrollPcrNoUpdate = conf.Butane(`---
+variant: flatcar
+version: 1.0.0
+storage:
+  files:
+    - path: /etc/flatcar/update.conf
+      overwrite: true
+      contents:
+        inline: |
+          SERVER=disabled
+  luks:
+  - name: rootencrypted
+    wipe_volume: true
+    device: "/dev/disk/by-partlabel/ROOT"
```

Comment:
Do these lines ought to be indented further by two more spaces?

Reply:
Yes, can do it, YAML is weird

Reply:
Fixed it for the existing test, too